keyboards/handwired/dactyl_manuform/5x8/readme.md

```diff
@@ -0,0 +1,27 @@
+# handwired/dactyl_manuform/5x8
+
+![handwired/dactyl_manuform/5x8](imgur.com image replace me!)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [iliorik](https://github.com/iliorik)
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
```

Comment:
?

Reply:
Hello. Thank you for comments to my code. I will fix my files and i will run on my keyboard for checking codes. If code will be work i will upload codes and push request for your control.

Reply:
I change readme.md

Reply:
i fix code
`![Dactyl_manuform 5x8](https://i.imgur.com/DtawD9r.jpeg)

* Keyboard Maintainer: [iliorik](https://github.com/iliorik)

I made this keyboard so that the buttons on the right side of the right half would replicate the standard keyboard.
I put macros on the left buttons of the left side keyboard for rec and play keypresses`